### DISCUSSION TITLE
Wallet TX Data Parsing for Miners

### DISCUSSION THREAD

**Discussion by marctheshark3** [nautls/nautilus-wallet#2] [Ideas]
I have built some tools that utilize the csv output from Yoroi. My idea is to implement this directly in the Nautilus app for each user with a wallet with the ability to view the summary statistics on their TX and other metrics. My purpose for doing this is cause I am a miner and want to know monthly minted coins, the price of coin it was mined at, and other things as well.

 This sort of thing would be very useful to other miners and could attract more to secure the network. Are you guys interested in collaborating and getting something like this into the wallet eventually?

**Comment by crystoll** [comment]
Actually quite the similar need here, with Yoroi, the remaining feature I need there, is to be able to output monthly mining, as well as any airdrops, and accumulate them monthly, for the taxes. Because nobody wants to handle FIFO per transaction. I was looking for some kind of csv export button, if Nautilus had that it would be game over for any other wallets for me :)
